portal-api/src/v1/gef/routes.js

```diff
@@ -126,4 +127,14 @@ router
   )
   .get(getAmendment);
 
+router
+  .route('/facilities/:facilityId/amendments')
+  .put(
+    validatePortalFacilityAmendmentsEnabled,
+    validateUserHasAtLeastOneAllowedRole({ allowedRoles: [MAKER] }),
+    mongoIdValidation('facilityId'),
+    handleExpressValidatorResult,
+    putAmendment,
+  );
+
```

Comment:
Do we have any payload validation in portal-api? we're already validating this in dtfs-central but might be nice to do it here too

Reply:
Now added😎